gitpod-io/gitpod
# Replace UA logo

## Description

This will replace the UA logo and add a message in the bottom right corner of the loading screen (workspace start).

- The placement is due to the already packed elements stacked vertically in the middle of the screen, which would cause the message to disappear on smaller viewports.
- The style was changed to be more subtle so that it's not conflicting with the existing information architecture and hierarchy of the workspace start page.
- Instead of the 🇺🇦 flag emoji, a subtle and generic custom SVG icon was used with the colors of the 🇺🇦 flag emoji, inspired from the dashboard status indicator for workspaces, integrations, and more.

See also [relevant discussion](https://gitpod.slack.com/archives/C019K7PND1A/p1651551163466379) (internal).

## Related Issue(s)
Fixes #10088 #8484

## How to test
Open a new workspace and notice the logo replacement as well as the message on in the bottom right corner of the loading screen (workspace start). 🇺🇦 

## Screenshots

| BEFORE | AFTER |
|-|-|
| <img width="1440" alt="Screenshot 2022-05-19 at 12 19 34 AM (2)" src="https://user-images.githubusercontent.com/120486/169158223-2f8abe37-42bb-4010-81d4-f1864d6bd3cd.png"> | <img width="1440" alt="Screenshot 2022-05-19 at 12 18 56 AM (2)" src="https://user-images.githubusercontent.com/120486/169158239-48135723-7c12-4a59-8f7a-4708f4a4375a.png"> |
| <img width="1440" alt="Screenshot 2022-05-19 at 12 19 29 AM (2)" src="https://user-images.githubusercontent.com/120486/169158278-3367f734-6fbe-4c05-a771-04b645adeadb.png"> | <img width="1440" alt="Screenshot 2022-05-19 at 12 18 46 AM (2)" src="https://user-images.githubusercontent.com/120486/169158292-aea60980-1c63-40e7-968f-dbb2c06ac9f5.png"> |

Alternatively, we could replace the custom SVG icon with the 🇺🇦 flag emoji.

| Example with the 🇺🇦  flag emoji (Light) | Example with the 🇺🇦  flag emoji (Dark) |
|-|-|
| <img width="1440" alt="Screenshot 2022-05-19 at 12 25 48 AM (2)" src="https://user-images.githubusercontent.com/120486/169158532-3c36cf36-4740-4110-9386-0e8b1a7e6177.png"> | <img width="1440" alt="Screenshot 2022-05-19 at 12 25 53 AM (2)" src="https://user-images.githubusercontent.com/120486/169158554-46799248-b7cf-4ded-ab21-cacd35bb67a8.png"> |

## Release Notes

```release-note
NONE
```

Cc @chrifro @atduarte @ghuntley because https://github.com/gitpod-io/gitpod/issues/10088
Cc @jldec because https://github.com/gitpod-io/gitpod/issues/8484
